contracts/cairo/src/escrow.cairo

```diff
@@ -145,10 +145,10 @@ mod Escrow {
         }
 
         fn cancel_order(ref self: ContractState, order_id: u256) {
-            assert(!self.orders_used.read(order_id), 'Order already withdrawed');
+            assert(!self.orders_used.read(order_id), 'Order already withdrew');
```

Comment:
```suggestion
            assert(!self.orders_used.read(order_id), 'Order withdrew or nonexistent);
```
We should add also that this can fail if the order_id doesn't exist